### PR TITLE
add template for PR

### PR DESCRIPTION
**Pull request by kazumura**
Signed-off-by: Kenji Kazumura <kzr@fujitsu.com>

As we discussed here "https://github.com/eclipse-ee4j/glassfish/pull/22945", I add PR template.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[//]: # "Please remove these comments"
+
+[//]: For general information, please read <"https://github.com/eclipse-ee4j/glassfish/blob/master/CONTRIBUTING.md">.
+
+[//]: If you want to create a PR for gh-pages, please read <"https://github.com/eclipse-ee4j/glassfish/blob/master/docs/README.md">.
```
Comment (bshannon) [inline .github/pull_request_template.md]: I would rephrase this as...

*DO NOT* create a PR for the gh-pages branch.  Instead, please read [the docs directory README file](https://github.com/eclipse-ee4j/glassfish/blob/master/docs/README.md) and create a PR for the appropriate docs subdirectory in the master branch.

**Review by bshannon — APPROVED** [review]
One minor comment, take it or leave it.

File: .github/pull_request_template.md (hunk 1 above)
Comment (kazumura) [reply .github/pull_request_template.md]: done

**Review by arjantijms — APPROVED** [review]
